python/ray/data/datasource/tfrecords_datasource.py

```diff
@@ -48,6 +74,25 @@ def _convert_example_to_dict(
     return record
 
 
+def _convert_arrow_to_examples(
+    arrow: "pyarrow.Table",
```

Comment:
Nit: `table` would probably be a better name, i.e. using the abstraction name rather than the library.
```suggestion
    table: "pyarrow.Table",
```

Reply:
+1 on this renaming